mitmproxy/addons/view.py

```diff
@@ -541,14 +545,10 @@ def inbounds(self, index: int) -> bool:
     # Event handlers
     def configure(self, updated):
         if "view_filter" in updated:
-            filt = None
             if ctx.options.view_filter:
-                filt = flowfilter.parse(ctx.options.view_filter)
-                if not filt:
-                    raise exceptions.OptionsError(
-                        "Invalid interception filter: %s" % ctx.options.view_filter
-                    )
```

Comment:
We should keep validating the filter when it is updated, this makes option validation more straightforward.

Reply:
I don't really get what you mean by it makes option validation more straightforward.